packages/features/ee/teams/components/MemberList.tsx

```diff
@@ -377,6 +378,24 @@ export default function MemberList(props: Props) {
           return filterValue.includes(rows.getValue(id));
         },
       },
+      {
+        id: "lastLogin",
+        accessorFn: (data) => data.lastLogin,
+        header: "Last Login",
+        cell: ({ row }) => {
+          const { lastLogin } = row.original;
+          return (
+            <div>
+              {lastLogin
+                ? formatLocalizedDateTime(lastLogin, {
+                    dateStyle: "full",
+                    timeStyle: "long",
+                  })
+                : "N/A"}
```

Comment:
The current implementation doesn't fall back to the active logged-in user, so for most users, it shows N/A. This doesn’t look optimal🤔

Reply:
Isn't that expected because we are storing the last logged in value in the DB and most of the users will not have the value until they re-login. Do you know of any other column using which we can deduce the last logged in time? Or any other idea to get the value?